ExpenseTracker/ExpenseTracker/Controllers/CategoriesController.cs

```diff
@@ -28,7 +28,7 @@ public IActionResult Details(int? id)
     {
         if (id == null)
         {
-            return NotFound();
+            return View("NotFound");
```

Comment:
Did you test it?

Reply:
It might not work since

- Views
  - Category
 does not have _NotFound_ view

Reply:
fixed.